tests/models/unit.py

```diff
@@ -202,3 +205,27 @@ def test_add_suggestion(af_tutorial_po, system):
     assert sugg is not None
     assert added
     assert len(untranslated_unit.get_suggestions()) == 1
+
+
+@pytest.mark.django_db
+def test_accept_suggestion(it_tutorial_po, system):
```

Comment:
Probably `test_accept_suggestion_update_wordcount` is more expressive to what you're actually testing. The same applies to the docstring below.